utils/cleanupResources.ts

```diff
@@ -0,0 +1,26 @@
+var dotenv = require('dotenv');
+dotenv.config();
+
+for (const envVar of ['PINECONE_ENVIRONMENT', 'PINECONE_API_KEY']) {
```

Comment:
**question:** This may be a minor issue, but if you run this locally outside CI where we're using secrets, I'd imagine this will just wipe things for whatever environment values you have configured, would that be an issue? Not sure if there's a good way around that as we don't want to embed the integration testing stuff in the repo itself.

Reply:
Yeah, it would run for whatever environment you are currently pointing at in your `.env`. Not ideal I suppose if you have a lot of test index data you're worried about accidentally deleting, but how likely are you to accidentally run `npm run integration:test:cleanup` from local? By wrapping this into a github actions workflow, we shouldn't ever really be running this locally.

Reply:
I suppose I could remove the `dotenv` invocation since that is only used locally. But I actually do want the ability to run this locally sometimes when developing new tests that throw off a lot of unintended indexes.

I think a more advanced version of this workflow would have us adopt a standardized naming scheme for indexes and collections (e.g. one that starts with a build number), and then trigger a cleanup job immediately after every CI run that deletes resources tied to that build number only. But that's more work / somewhat error prone. This quick and dirty approach gives most of the benefits.

Reply:
Yeah I ultimately think this is fine, it would be very unlikely somewhat would run the integration cleanup step by accident or without knowing what it does.